forestserviceprototype/specialuseform/templates/specialuseform/submitted_permit.html

```diff
@@ -1,8 +1,24 @@
 {% extends "specialuseform/base.html" %}
 
 {% block content %}
-  <h1>Thank you!</h1>
+  <h1>Application Submitted</h1>
   <p class="usa-font-lead">
-    We've got it! We'll be in touch.
+    Application #{{ permit.permit_id }}: {{ permit.event_name }}
   </p>
+
+  <p> Permit Status: {{ permit.get_status_display }}</p>
+
+  <p>You will recieve an email notification from the Forest Service in a few days about whether your permit has been approved.
```

Comment:
What can we can tell them to do if they have questions in the meantime or if they don't hear back?